apps/web/src/views/Farms/components/FarmCard/V3/FarmV3ApyButton.tsx

```diff
@@ -32,9 +34,8 @@ import { useFarmsV3Public } from 'state/farmsV3/hooks'
 import { Field } from 'state/mint/actions'
 import LiquidityFormProvider from 'views/AddLiquidityV3/formViews/V3FormView/form/LiquidityFormProvider'
 import { useV3FormState } from 'views/AddLiquidityV3/formViews/V3FormView/form/reducer'
-import { V3Farm } from 'views/Farms/FarmsV3'
 import { FarmsV3Context } from 'views/Farms'
-import isUndefinedOrNull from '@pancakeswap/utils/isUndefinedOrNull'
+import { V3Farm } from 'views/Farms/FarmsV3'
```

Comment:
Please be aware of the dependency relationship. It may cause circular references